docs/docs/working-with-images-in-markdown.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Working With Images in Markdown
+---
+
+When building Gatsby sites composed primarily of markdown pages, insertion of images can enhance the content around it. Adding images can be done in multiple ways.
+
+## Featured images with Frontmatter fields
+
+In sites like a blog, you may want to include a featured image that appears at the top of a page. One way to do this can be by grabbing the image filename from a frontmatter field and then transforming it with `gatsby-plugin-sharp` in a GraphQL query.
+
+It is assumed you already have programatically generated pages from markdown using renderers like `gatsby-transformer-remark` or `gatsby-plugin-mdx`. If not, take a read through up to [Part 7 of the Gatsby Tutorial](tutorial/part-seven/). This will build upon the tutorial and as such, `gatsby-transformer-remark` will be used for this example
```

Comment:
```suggestion
This solution assumes you already have programmatically generated pages from Markdown with renderers like `gatsby-transformer-remark` or `gatsby-plugin-mdx`. If not, take a read through up to [Part 7 of the Gatsby Tutorial](tutorial/part-seven/). This will build upon the tutorial and as such, `gatsby-transformer-remark` will be used for this example.
```